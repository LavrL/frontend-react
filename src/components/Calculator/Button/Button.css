.btn {
    height: 50px;
    width: 50px;
    border-radius: 50%;
    display: flex;
    align-items: center;
    text-align: center;
    justify-content: center;
    background-color: grey;
}
.btn-orange {
    background-color: orange;
}
