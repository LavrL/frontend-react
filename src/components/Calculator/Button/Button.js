import React from 'react';
import './Button.css';

export const Button = (props) => {
    return (
        <div className={props.buttonClass}
            onClick={props.onClick}>
            {props.value}
        </div>
    )
}
export default Button;