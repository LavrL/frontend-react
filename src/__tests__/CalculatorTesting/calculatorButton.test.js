import React from 'react';
import renderer from 'react-test-renderer';
import { Button } from '../../components/Calculator/Button/Button';

describe('Button', () => {
    test('Calculator Button snapshot', () => {
        const component = renderer.create(<Button className={"btn"}
            onClick={"onClick"}>{"Save"}</Button>);
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
    
})