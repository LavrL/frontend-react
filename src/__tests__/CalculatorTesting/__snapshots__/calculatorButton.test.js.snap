// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button Calculator Button snapshot 1`] = `
<div
  onClick="onClick"
/>
`;
